frontend/app/styles/main.css

```diff
@@ -1,6 +1,9 @@
 /* Required padding for .navbar-fixed-top is 50px. Add to top level container div if needed.
  * Remove if using .navbar-static-top. Change if height of navigation changes. */
-
+body {
+    background-color: #000 !important;
```

Comment:
don't ever use `!important`  in your CSS if you can avoid it. it's often an easy way to solve annoying style problems, but it's a very bad habit that will get you into trouble quickly